common/changes/office-ui-fabric-react/v-vibr-DisabledDropown_2018-09-25-00-59.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Dropdown: prevent dropdown to open on keyboard events when in disabled mode.",
+      "type": "minor"
```

Comment:
should be patch as no API has changed... (just wondering - why did you use `minor`?)

Reply:
I think I confused which one is what))